loops-arrays-objects.js

```diff
@@ -31,8 +33,21 @@ const scores = [
 // A
 // D
 // ==========================================
+console.log(scores.score);
 
-
+for (let i = 0; i < scores.length; i++) {
+    if (scores[i].score < 60) {
+        console.log("F");
+    } else if (scores[i].score < 70 && scores[i].score >= 60) {
+        console.log("D");
+    } else if (scores[i].score < 80 && scores[i].score >= 70) {
+        console.log("C");
+    } else if (scores[i].score < 90 && scores[i].score >= 80) {
+        console.log("B");
+    } else if (scores[i].score < 100 && scores[i].score >= 90) {
```

Comment:
```suggestion
    } else if (scores[i].score < 70) {
        console.log("D");
    } else if (scores[i].score < 80) {
        console.log("C");
    } else if (scores[i].score < 90) {
        console.log("B");
    } else if (scores[i].score < 100) {
```
Omdat telkens de voorgaande if false heeft opgeleverd, hoef je niet meer te controleren of het getal groter is dan de if daarvoor